lib/ui/painting.dart

```diff
@@ -399,6 +423,7 @@ class Paint {
     _data.setInt32(_kColorOffset, encoded, _kFakeHostEndian);
   }
 
+  // Must be kept in sync with the default in paint.cc
```

Comment:
LOL.  I also found another one, just to spare you the task of commenting on _each_ and _every_ one.  [Sorry, I should have done a search the first time.]